tom_lt/lt.py

```diff
@@ -16,27 +17,35 @@
 from tom_observations.facility import GenericObservationForm, GenericObservationFacility
 from tom_targets.models import Target
 
-# Determine settings for this module
+
+"""
+TOM Toolkit LT Module, Version 0.3.0
+
+For required steps and isntructions to use this module please see https://github.com/TOMToolkit/tom_lt/blob
+/master/README.md
+"""
 try:
-    LT_SETTINGS = settings.FACILITIES['LT']
-except (AttributeError, KeyError):
+    import tom_lt.secret
```

Comment:
Again, for consistency's sake I think the tom_lt credentials should follow the same format as the other facilities. I'm not entirely sure why the LT module would necessitate its own pattern. I think the settings should go where the others do, in settings.py within the dictionary called `FACILITIES`.